backend/package-lock.json

```diff
@@ -1,3713 +1,6 @@
 {
```

Comment:
What version is your node & npm at? They should be at vesions 16 & 8, respectively. If you could run `npm i` on the frontend and backend after you update them, it should hopefully get rid of these big changes in the package.json